MapboxNavigation/FeedbackViewController.swift

```diff
@@ -35,6 +34,11 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
     
     let interactor = Interactor()
     
+    /**
+     The feedback items that are visible and selectable by the user.
+     */
+    public var sections: [[FeedbackItem]] = [[.turnNotAllowed, .closure, .reportTraffic, .confusingInstructions, .generalMapError, .badRoute]]
```

Comment:
Why is this an array nested inside an array? What does the outer array represent?

Reply:
Nested sections were previously used to forcefully use one or two rows based on device orientation. This is being handled automatically now:
https://github.com/mapbox/mapbox-navigation-ios/blob/4fa51577668b46700bbd07ff37f0c45b636427f4/MapboxNavigation/FeedbackViewController.swift#L287-L289

So should be fine to flatten `[[FeedbackItem]]` to `[FeedbackItem]` and always use the first section. However, keeping the current structure is a bit more versatile.